layouts/program-committee/section.html

```diff
@@ -0,0 +1,101 @@
+{{ define "main" }}
+<div class="margin-bottom-30">
+    <!-- Committee member -->
```

Comment:
Leveraging either Data or Lists functionalities will prevent us from writing all this html.